make/photon/prepare/templates/jobservice/config.yml.jinja

```diff
@@ -51,3 +51,6 @@ metric:
   path: {{ metric.path }}
   port: {{ metric.port }}
 {% endif %}
+
+# the max size of job log returned by API, default is 10M
+max_retrieve_size_mb: 10
```

Comment:
Would it be possible to use the value '-1' to represent an unlimited scenario?

Reply:
<=0 will be considerd to unlimited, -1 is included.